src/pages/variants/VariantTable.tsx

```diff
@@ -10,26 +10,34 @@ import {
   Frequencies,
   SelectedSuggestion,
 } from 'store/graphql/variants/models';
-import { setSqons } from 'store/actionCreators/virtualStudies';
 import { connect, ConnectedProps } from 'react-redux';
 import { DispatchVirtualStudies } from 'store/virtualStudiesTypes';
 import { Sqon } from 'store/sqon';
 import { withHistory } from 'services/history';
 // @ts-ignore
 import { compose } from 'recompose';
 import { RouteComponentProps } from 'react-router-dom';
-import { generateKfIdsSqon } from 'store/sqonUtils';
 import ROUTES from 'common/routes';
+import { createQueryInCohortBuilder } from '../../store/actionCreators/studyPage';
+import { RootState } from '../../store/rootState';
+import { addToSqons } from '../../common/sqonUtils';
 
 const DEFAULT_PAGE_NUM = 1;
+type VariantTableState = {
+  currentVirtualStudy: Sqon[];
+};
 
 const isEven = (n: number) => n % 2 === 0;
 
 const mapDispatch = (dispatch: DispatchVirtualStudies) => ({
-  setSqons: (sqons: Sqon | Sqon[]) => dispatch(setSqons(sqons)),
+  onClickParticipantLink: (sqons: Sqon[]) => dispatch(createQueryInCohortBuilder(sqons)),
+});
+
+const mapState = (state: RootState): VariantTableState => ({
+  currentVirtualStudy: state.currentVirtualStudy.sqons,
```

Comment:
I think `currentSqons` would be more approriate than `currentVirtualStudy` in `mapState`